python/setup.py

```diff
@@ -72,9 +69,6 @@
     keywords="pynessie",
     name="pynessie",
     packages=find_packages(include=["pynessie", "pynessie.*"]),
-    setup_requires=setup_requirements,
-    test_suite="tests",
-    tests_require=[],
```

Comment:
running tests through setup.py is an outdated mechanism, see deprecation notice here:
https://pypi.org/project/pytest-runner/

we are using tox for that as recommended